data-structures-queues/take-2nd.js

```diff
@@ -0,0 +1,17 @@
+/* exported take2nd */
+
+function take2nd(queue) {
+  const get = queue.dequeue();
+  if (get === undefined) {
+    return get;
+  }
+  const get2 = queue.peek();
+  if (get2 === undefined) {
+    return get;
+  }
+  queue.dequeue(get2);
```

Comment:
The `dequeue` method does not expect any parameters, so you shouldn't provide any arguments:

---

![image](https://user-images.githubusercontent.com/16143771/183978412-a38fd8b2-0fc0-4c70-b871-5da490ac46f0.png)